### PR TITLE
Try to clarify MRC support

### PR DESCRIPTION
**Pull request by melissalinkert**
See #294.

This at least adds links to the CCP-EM specifications; happy to hear other suggestions.

### CODE REVIEW

File: src/main/resources/format-pages.txt [hunk 1]
```diff
@@ -1678,10 +1678,12 @@ reader = MolecularImagingReader
 [MRC (Medical Research Council)]
 pagename = mrc
 extensions = .mrc, .st, .ali, .map, .rec, .mrcs
-developer = `MRC Laboratory of Molecular Biology <https://www2.mrc-lmb.cam.ac.uk/>`_
+developer = * `MRC Laboratory of Molecular Biology <https://www2.mrc-lmb.cam.ac.uk/>`_ \n
```
Comment (sbesson) [inline src/main/resources/format-pages.txt:1681]: See https://bio-formats--297.org.readthedocs.build/en/297/formats/mrc.html. Looks like mutliple developers are not rendered correctly. Maybe we should mark `MRC` as `(formerly MRC Laboratory...)` as the ownership/maintenance seems to have been fully transferred to CCP-EM

**Review by sbesson — CHANGES_REQUESTED** [review]

File: src/main/resources/format-pages.txt (hunk 1 above)
Comment (melissalinkert) [reply src/main/resources/format-pages.txt:1681]: Multiple developers should now be rendered correctly.

**Review by sbesson — APPROVED** [review]
https://bio-formats--297.org.readthedocs.build/en/297/formats/mrc.html looks good to me

**Review by dgault — APPROVED** [review]
Changes look good to me

**Pull request merged**